ibis/bigquery/client.py

```diff
@@ -369,8 +424,9 @@ def list_tables(self, like=None, database=None):
         return result
 
     def get_schema(self, name, database=None):
-        table_id, dataset_id = _ensure_split(name, database)
-        table_ref = self.client.dataset(dataset_id).table(table_id)
+        table_ref = self.client.dataset(
+            database or self.dataset_id
```

Comment:
Seems like database should be optionally split into project_id and dataset_id, too. (Here and other methods.)

Reply:
Assuming that `self.client`'s project is the one we're getting data from is correct, then we don't need to split here. We *should* enforce that assumption though, through verifying that the project name isn't included in the database or it's equivalent to `self.project_id`.

Reply:
What if someone wants to write a query that does a cross-project join, like joining a public dataset with their own dataset?

Reply:
Ah, I was wondering if you'd ask that :) This is actually enforced generically in ibis, you can't have multiple clients underlying the same expression. I made an issue about this (#1429). It's marked as "Future", but if missing this functionality is a non-starter for you then we can definitely push up the priority.

Reply:
Ah, well the 1 client restriction does kind of correlate with my thoughts that the default on the client is usually the one you are charging to. You couldn't do a cross-client query that tries to charge the query to two different projects (or I guess you could, but it'd be ambiguous which one gets charged).